ts/src/base/Exchange.ts

```diff
@@ -5982,6 +5982,14 @@ export default class Exchange {
     parseGreeks (greeks, market: Market = undefined): Greeks {
         throw new NotSupported (this.id + ' parseGreeks () is not supported yet');
     }
+
+    handleNetworkParameter (code: string, params = {}) {
```

Comment:
As is `handleNetworkParameter` is not very useful, it should be using `networkCodeToId` instead

Reply:
> As is `handleNetworkParameter` is not very useful, it should be using `networkCodeToId` if network is available

`networkCodeToId` has a completely different purpose, it gets the exchange specific networkId based on the CCXT network code. `handleNetworkParameter` provides a default network code from `options` if a `networkCode` is not provided in params